src/Components/Shared/ViewDailyReport/index.js

```diff
@@ -1,115 +1,250 @@
+          return (
+            <div key={key}>
+              <ReportCard {...reportCardData[key]}>
+                {reportCardData[key].children}
+              </ReportCard>
+            </div>
+          );
+        }
+        return null;
+      })} */}
+      <ReportCard
+        title={t('patient.report.didYouTake')}
+        titleIcon={<PillIcon />}
+        color={medicationWasTaken ? 'green' : 'red'}
+        statusIcon={medicationWasTaken ? CheckCircleIcon : HighlightOffIcon}
+        statusText={
+          medicationWasTaken ? t('commonWords.yes') : t('commonWords.no')
+        }
+      />
+      <ReportCard
+        title={t('patient.report.symptomsTitle')}
+        titleIcon={<Assignment />}
+        color={hadSymptoms ? 'red' : 'green'}
+        statusIcon={hadSymptoms ? HighlightOffIcon : CheckCircleIcon}
+        statusText={
+          hadSymptoms
+            ? t('coordinator.sideBar.symptomsSince')
+            : t('coordinator.recentReports.noSymptoms')
+        }
+      >
+        {hadSymptoms && <SymptomList symptoms={symptoms} />}
+      </ReportCard>
+      <ReportCard
+        title={t('patient.report.moodTitle')}
+        titleIcon={<InsertEmoticon />}
+        color={doingOkay ? 'green' : 'red'}
+        statusIcon={
+          doingOkay ? SentimentVerySatisfied : SentimentVeryDissatisfied
+        }
+        statusText={
+          doingOkay
+            ? t('patient.report.doingWell')
+            : t('patient.report.needSupport')
+        }
+      >
+        {doingOkayReason && <Typography>{doingOkayReason}</Typography>}
+      </ReportCard>
+      <ReportCard
+        title={t('commonWords.stripPhoto')}
+        titleIcon={<CameraIcon />}
+        color="blue"
+        statusIcon={photoWasRequired ? CheckCircleIcon : RemoveCircleIcon}
+        statusText={
+          photoWasRequired ? '' : t('patient.report.confirmation.noPhoto')
+        }
+      >
+        {photoWasRequired && (
+          <>
+            {photoUrl ? (
+              <>
+                <ZoomableImage
+                  initialScale={0.5}
+                  maxHeight="200px"
+                  url={photoUrl}
+                />
+              </>
+            ) : (
+              <>
+                <Status
+                  text={t('report.missedPhoto')}
+                  icon={HighlightOffIcon}
+                  color={Colors.red}
+                />
                 <Seperator />
-                <SymptomList symptoms={symptoms} />
-            </>}
-        </ExpandableCard>
-
-        <ExpandableCard hideToggle title={t('patient.report.moodTitle')} icon={InsertEmoticon}>
+                <Typography>{t('patient.report.photo.whyUnable')}</Typography>
+                <AnswerText>
+                  {whyPhotoWasSkipped || t('coordinator.sideBar.noReason')}
+                </AnswerText>
+              </>
+            )}
+          </>
+        )}
+      </ReportCard>
+
+      {/* <InputCard>
```

Comment:
Why all the commented out code here?

Not a super big deal but might lead to confusion